src/content/resources/javascript/es6/arrow-functions.md

```diff
@@ -40,12 +41,27 @@ const sayHello = name => {
 };
 ```
 
-Another interesting feature that also reduces the syntax within arrow functions, is that if a function only returns an expression using one line of code, you can omit the `return` keyword and the curly braces. In arrow functions without braces, the `return` is implicit, meaning you don't need to include the `return` keyword. The following function returns "hello":
+Another interesting feature that also reduces the syntax within arrow functions, is that if a function only returns an expression without any 'business logic', you can omit the `return` keyword and the curly braces. In arrow functions without braces, the `return` is implicit, meaning you don't need to include the `return` keyword. The following function returns "hello":
 
 ```js
 const sayHello = () => "hello";
 ```
 
+The same is also true of functions returning an expression using parameters, too.
+
+```js
+const sayHello = name => `hello ${name}`;
+```
+
+Or even...
```

Comment:
Might want to mention what this actually does.

It's using the parser precedence to wrap an object expression inside a grouped expression thus resulting in returning an object instead of expanding into a function body

Reply:
I put it in now. Added a little bit of humour/light jokeyness to it as per my typical writing style

Reply:
(and yes, I probably just made up a word)